homeassistant/components/media_player/yamaha.py

```diff
@@ -22,22 +22,30 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Yamaha platform."""
     import rxv
-    add_devices(YamahaDevice(config.get("name"), receiver)
+
+    excludes = config.get('excludes', None)
```

Comment:
Can you extract the values `excludes` and `mapping` into constants at the top of the file in the format `CONF_MAPPING = 'mapping'`.

I also think that the names mapping and excludes do not properly represent the meaning of the configuration option. Something like `source_name` and `source_ignore` would for example be more fitting.